SPRINTLOG.md

```diff
@@ -450,3 +450,7 @@ _Nothing merged during this sprint_
 - Workflow: Bump GitHub checkout action to v4 ([#1556](https://github.com/ScilifelabDataCentre/dds_web/pull/1556))
 - Workflow: CodeQL action version(s) bumped to v3 ([#1569](https://github.com/ScilifelabDataCentre/dds_web/pull/1569))
 - Workflow: Setup-node, codecov and upload-sarif action versions bumped to v4, v4 and v3, respectively ([#1570](https://github.com/ScilifelabDataCentre/dds_web/pull/1570))
+
+# 2024-11-04 - 2024-11-15
+
+- Change the errror when trying to download data after a password reset to an AuthenticationError to avoid getting an alert ([#1571](https://github.com/ScilifelabDataCentre/dds_web/pull/1571))
```

Comment:
It's not like the exception is removed. AccessDeniedErrror is also an exception.

What I found is that, when the exception/error is a 500, it is always logged in the backend, probably due to them being sever-side errors and considered important to check.

However, access dennied is a 400 error, in this case nothing is logged, because it is an user error and nothing to debug.


This is also the reason why the solution for MaintenanceError is to filter out the logs, because maintenance is, by definition, a 500 error. So I assumed it was better to filter out than to change it to a generic 400 error.
In this case, I assumed access dennied error to be a proper response code to this error

Reply:
That's right actually, then you can just fix the typo and reformulate, something like:

```suggestion
- Change the error raised upon attempt to download data after a password reset to an AuthenticationError to avoid getting an alert ([#1571](https://github.com/ScilifelabDataCentre/dds_web/pull/1571))
```